docs/quick-launch.md

```diff
@@ -0,0 +1,125 @@
+```
+
+In the above example, `-qs` is the flag that indicates to the RSPeer client that
+a quick start configuration is being supplied. The long string of gibberish that
+follows is the Base64-encoded configuration. *nix and WSL users can decode this
+with the following command:
+
+```
+base64 -d <<< 'ewogICJSc1VzZXJuYW1lIjogInVzZXJAZXhhbXBsZS5jb20iLAogICJSc1Bhc3N3b3JkIjogInBAc3N3MHJkISIsCiAgIldvcmxkIjogMTMzNywKICAiU2NyaXB0TmFtZSI6ICJDb29sIFNjcmlwdCIsCiAgIklzUmVwb1NjcmlwdCI6IHRydWUsCiAgIlNjcmlwdEFyZ3MiOiAiIiwKICAiVXNlUHJveHkiOiB0cnVlLAogICJQcm94eVBvcnQiOiAiODA4MCIsCiAgIlByb3h5SXAiOiAiMTI3LjAuMC4xIiwKICAiUHJveHlVc2VyIjogIiIsCiAgIlByb3h5UGFzcyI6ICIiLAogICJDb25maWciOiB7CiAgICAiTG93Q3B1TW9kZSI6IGZhbHNlLAogICAgIlN1cGVyTG93Q3B1TW9kZSI6IGZhbHNlLAogICAgIkRpc2FibGVNb2RlbFJlbmRlcmluZyI6IGZhbHNlLAogICAgIkRpc2FibGVTY2VuZVJlbmRlcmluZyI6IGZhbHNlCiAgfQp9Cg=='
+```
+
+There are also [tools](http://www.utilities-online.info/base64/) available
+online for handling Base64. You should see the following result:
+
+```json
+{
+  "RsUsername": "user@example.com",
+  "RsPassword": "p@ssw0rd!",
+  "World": 1337,
+  "ScriptName": "Cool Script",
+  "IsRepoScript": true,
+  "ScriptArgs": "",
+  "UseProxy": true,
+  "ProxyPort": "8080",
+  "ProxyIp": "127.0.0.1",
+  "ProxyUser": "",
+  "ProxyPass": "",
+  "Config": {
+    "LowCpuMode": false,
+    "SuperLowCpuMode": false,
+    "DisableModelRendering": false,
+    "DisableSceneRendering": false
+  }
+}
+```
+
+This is the same exact format written by the QuickLaunch Generator (see
+[Basic Usage](#basic-usage)).
+
+## Reference
+
+| **Name** | **Date Type** | **Description** |
+| -------- | ------------- | --------------- |
+| `RsUsername` | *string* | Login name or email of the Runescape account. |
+| `RsPassword` | *string* | Login password of the Runescape account. |
+| `World` | *integer* | World to join (use `-1` or `null` to pick any world). |
+| `ScriptName` | *string* | Name of the script to start. |
+| `IsRepoScript` | *boolean* | `true` if the script should be executed from the SDN, `false` if it's available locally. |
+| `ScriptArgs` | *string* | Custom arguments to pass to the script on startup. |
+| `UseProxy` | *boolean* | `true` to connect via proxy. |
+| `ProxyIp` | *string* | IP address of the proxy. |
+| `ProxyPort` | *string* | Port of the proxy. |
+| `ProxyUser` | *string* | Username for proxy authentication. |
+| `ProxyPass` | *string* | Password for proxy authentication. |
+| `Config` | *object* | Additional client configuration, see [Config](#config) below. |
+
+### Config
+
+| **Name** | **Date Type** | **Description** |
+| -------- | ------------- | --------------- |
+| `LowCpuMode` | *boolean* | `true` to enable low CPU mode. |
+| `SuperLowCpuMode` | *boolean* | `true` to enable *super* low CPU mode. |
+| `EngineTickDelay` | *integer* | Delays  (`SuperLowCpuMode` sets this to `5`). |
```

Comment:
I'd love to know more about what exactly this `EngineTickDelay` influences. Is this something to do with rendering or is it more to do with the polling frequency of script loops?

Reply:
Hi, this slows the game down.

Reply:
Got it, so that would be:

```suggestion
| `EngineTickDelay` | *integer* | Slow the game down (`SuperLowCpuMode` sets this to `5`). |
```

Now that I think about it, I'm not sure where I read that `SuperLowCpuMode` sets this option. I can't verify this now so I'm not too confident that I'm right in saying that it gets set to `5`. 🤷‍♀